output.tf

```diff
@@ -1,19 +1,19 @@
 output "key_arn" {
-  value       = aws_kms_key.default.arn
+  value       = join("", aws_kms_key.default.*.arn)
   description = "Key ARN"
 }
 
 output "key_id" {
-  value       = aws_kms_key.default.key_id
+  value       = join("", aws_kms_alias.default.*.id)
```

Comment:
```suggestion
  value       = join("", aws_kms_key.default.*.key_id)
```

Reply:
sorry, is my dyslexia , should be good now